tests/on_wlm/test_wlm_orc_config_settings.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from smartsim.error import SmartSimError
+
+# retrieved from pytest fixtures
+if pytest.test_launcher not in pytest.wlm_options:
+    pytestmark = pytest.mark.skip(reason="Not testing WLM integrations")
+
+try:
+    from smartredis import Client
+
+    config_setter = Client.config_set
+except AttributeError:
+    pytestmark = pytest.mark.skip(reason="SmartRedis version is < 0.3.0")
+
+
+def test_config_methods_on_wlm_cluster(dbutils, db_cluster):
+    """test setting clustered 3-node orchestrator configurations"""
```

Comment:
condense this test with the ``bad`` variant so we only spin up the database once.